float64 yaw
float64 roll
float64 pitch
float64 longitude
float64 latitude
float64 angle
string LongitudeSign
string LatitudeSign
